extractive_qa/extractive-question-answering.ipynb

```diff
@@ -938,23 +904,19 @@
    "id": "c048dfe1-0a0a-4b01-a9ee-79860827b970",
    "metadata": {},
    "source": [
-    "## Extracting an Answer\n",
-    "\n",
-    "Let's think of `extract_answer` as the **main** of this operation. You throw a question at it and it uses Qdrant, to pull out all the related context. But the function doesn't stop there!\n",
-    "\n",
-    "It puts all this related context into an \"Answer Extraction Machine\" — that's what we're calling the `reader` part. The reader looks at each piece of context, one by one, and tries to extract an answer from it that best fits your question. Then it stamps each of these answers with a score based on how good it thinks each answer is.\n",
+    "## Extracting an answer\n",
     "\n",
-    "Once all the context pieces have been squeezed through the Answer Extraction Machine, we end up with a list of potential answers, each with its own score. But hold on, we're not done yet!\n",
+    "Here is how the engine operates:\n",
     "\n",
-    "Now, we've got this pile of answers, but we need to know which one's the best, right? So, our function sorts all the answers based on their scores, with the top scorer at the front.\n",
+    "1. The central part of the function is `extract_answer`. Qdrant processes your question and retrieves all related context. \n",
     "\n",
-    "But we're not just gonna serve you the answers all alone. Nope, we also give you a bit of context to each answer. That's why we have the \"title\" added to the answer. \n",
+    "2. All related context is processed via the `reader`, which looks at each piece of context and extracts an answer that best fits your question. \n",
     "\n",
-    "Lastly, the function spits out the sorted list of answers, along with their scores and associated titles. This way, you don't just get a bare answer but also get to know where it came from and how confident the system is about the answer.\n",
+    "3. The function sorts all answers by confidence score, with the top score at the front. Each answer has a title in order to provide context. \n",
     "\n",
-    "And that's pretty much it! \n",
+    "4. The result is a sorted list of potential answers, their confidence scores and associated titles. \n",
     "\n",
-    "You pop in a question, and out comes a neatly ordered list of the best answers it could find, complete with a bit of background and a confidence score. It's like magic, but with a whole lot of Qdrant's vector search power behind it!"
+    "That's it! All you have to do is put in a question, and wait for an ordered list of the best possible answers. The advantage of this engine is that it also tells you where the answer came from and how confident the system is about its appropriateness."
```

Comment:
@davidmyriel `appropriateness` is a bit confusing for most readers with English as a second language. Can we try for something simpler e.g. confidence score?